website/docs/r/waf_rule.html.markdown

```diff
@@ -34,13 +34,12 @@ The following arguments are supported:
 The following attributes are exported:
 
 * `id` - The WAF Rule ID, the same as rule_id.
-* `zone_id` - The DNS zone ID.
 * `package_id` - The ID of the WAF Rule Package that contains the rule.
 
 ## Import
 
-Rules can be imported using a composite ID formed of zone name and the WAF Rule ID, e.g.
+Rules can be imported using a composite ID formed of zone ID and the WAF Rule ID, e.g.
 
 ```
-$ terraform import cloudflare_waf_rule.100000 example.com/100000
+$ terraform import cloudflare_waf_rule.100000 ae36f999674d196762efcc5abb06b345/100000
```

Comment:
Why not keep this as the only case where the zone name could be used ?
This is user-facing, and it's easier for an user to provide the zone name.
In the rest of the code, it's possible to use a `zone` resource or `zones` data to fetch the zone ID to use in other blocks. Running `terraform import` however is usually manual.

Reply:
Please see #227 and https://github.com/terraform-providers/terraform-provider-cloudflare/issues/161#issuecomment-441128985 for full details on why this has been done.

Reply:
I totally understand why in the code side (i.e. terraform resources and all), but the `terraform import` command is user-facing and should be easy to use. How can I grab the zone ID from the UI ? Do I need to do an API call to have access to it ? If so, then it's not adapted to `terraform import` !

I'm not arguing with the removal of the zone name elsewhere, just about the specific import method.

Reply:
The reasoning is identical to why it’s being deprecated for the internals; ambiguity in which zone name is intended to be used. 

The zone ID is available from the overview UI of a Cloudflare zone (right hand side navigation bar).